howtos/manage_shared_libraries/env_vars.rst

```diff
@@ -215,14 +166,12 @@ the environment variables poiting to the "lib" and "bin" folders.
     toolA:shared=True
 
     [generators]
-    virtualenv
+    virtualrunenv
 
-
-**In the terminal window:**
+In the terminal window:
 
 .. code-block:: bash
 
-    conan install .
-    source activate
-    toolA --someparams
-
+    $ conan install .
+    $ source activate_run
+    $ toolA --someparams
```

Comment:
I think the security restriction of OSX is also active here, and it might be ncessary to do the DYLD_LIBRARY_PATH = in the command line.